### ISSUE TITLE
"Upload JSON" within the OpenPose model breaks UI

### ISSUE THREAD

**Issue by NextDiffusion** [Mikubill/sd-webui-controlnet#2332]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits of both this extension and the webui

### What happened?

I just came across a break of the UI when selecting the OpenPose Control Type. After some digging i found that the breaking of the UI was caused by a font-size put on the "Upload JSON" input. Please remove the font size (100px), so the UI won't break when selecting the OpenPose Control Type. Thank you

### Steps to reproduce the problem

- Select the OpenPose Control Type
- Breaking of UI (The width of the UI suddenly becomes much wider and shows a scrollbar with white space on the right side)

### What should have happened?

Basically no overflow of white space

### Commit where the problem happens

webui: v1.6.1
controlnet: v1.1.422

### Command Line Arguments

```Shell
none
```

### List of enabled extensions

Doesn't matter here.

### Console logs

```Shell
none
```

### Additional information

Styling of the "Upload JSON" input breaks UI

**Comment by huchenlei** [comment]
Can you attach a screenshot of the broken UI?

**Comment by NextDiffusion** [comment]
Sure, below you will find two screenshots. One where you see the break of the UI with white space on the right. The other one is the inspector tool, where you can clearly see that the white space is caused by the wrong styling of the input "Upload JSON".
![UI_break_white_space_openpose_json_input](https://github.com/Mikubill/sd-webui-controlnet/assets/137604642/df204c53-bb2d-4e32-b2dd-5906c5b6e697)
![UI_break_white_space_openpose](https://github.com/Mikubill/sd-webui-controlnet/assets/137604642/5d5af3f5-6645-4287-953c-b3f1c5d8696a)
